### DISCUSSION TITLE
XLL Documentation say it applies to 2013. Not clear that it also works on 2016, 2019 and Subscription

### DISCUSSION THREAD

**Discussion by keyur32** [MicrosoftDocs/office-developer-client-docs#654] [Q&A]

**Comment by lindalu-MSFT** [comment]
Hi @keyur32 Should I change the Applies to: section in all the Excel documentation in docs/Escel to add "Excel 2019, Excel 2016, and Excel Online?" I can do that pretty easily with S&R.

**Comment by Buzzardbait** [comment]
I'm finding a lot of documentation is obsolete and should be removed as being a nuisance. For example, the tool InfoPath is brought up regarding creating Trusted Forms but no such tool exists in Office365. Right now, it appears that there is no way to add a WebSocket to an Excel form. The tool is rejected because the form is not trusted. But the method to make it trusted doesn't exist in Office365.

**Comment by facebookegypt** [comment]
I'm sorry, I've been searching a lot for dbversion value for MS Access 2016. Is it dbversion160 ?
